packages/keystatic/src/form/fields/array/ui.tsx

```diff
@@ -409,9 +397,7 @@ export function ArrayFieldListView<Element extends ComponentSchema>(
               >
                 <Icon src={trash2Icon} />
               </ActionButton>
-              <Tooltip tone="critical">
-                {stringFormatter.format('delete')}
-              </Tooltip>
+              <Tooltip>{stringFormatter.format('delete')}</Tooltip>
```

Comment:
Unnecessary critical tone; it's not a destructive action. Prefer "remove" but we don't have the l10n strings.